examples/play-from-disk/jsfiddle/demo.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-env browser */
+
+let pc = new RTCPeerConnection()
+let log = msg => {
+  document.getElementById('div').innerHTML += msg + '<br>'
+}
+
+pc.ontrack = function (event) {
+  var el = document.createElement(event.track.kind)
+  el.srcObject = event.streams[0]
+  el.autoplay = true
+  el.controls = true
+
+  document.getElementById('remoteVideos').appendChild(el)
+}
+
+pc.oniceconnectionstatechange = e => log(pc.iceConnectionState)
+pc.onicecandidate = event => {
+  if (event.candidate === null) {
+    document.getElementById('localSessionDescription').value = btoa(JSON.stringify(pc.localDescription))
+  }
+}
+
+// Offer to receive 1 audio, and 2 video tracks
+pc.addTransceiver('audio', {'direction': 'sendrecv'})
+pc.addTransceiver('video', {'direction': 'sendrecv'})
```

Comment:
Remove the extra audio and video transceivers.

Reply:
Fixed!